### PR TITLE
Add tholdoffset

### PR DESCRIPTION
**Pull request by Mariusthvdb**
Schermafbeelding 2020-02-01 om 00 49 14
to SML binary_sensor

### CODE REVIEW

**Comment by robmarkcole** [comment]
please rename `tholdoffset` to `threshold_offset`

**Comment by Mariusthvdb** [comment]
> please rename `tholdoffset` to `threshold_offset`

done, but can't resolve above conflict...can you?

**Comment by robmarkcole** [comment]
Please resolve conflicts and [rebase](https://developers.home-assistant.io/docs/en/development_catching_up.html)

File: custom_components/huesensor/binary_sensor.py [hunk 1]
```diff
@@ -34,6 +34,10 @@
         "reachable",
         "sensitivity",
         "threshold",
+        "threshold_offset"
+    ],
+    "RWL": ["last_updated", "battery", "on", "reachable"],
```
Comment (robmarkcole) [inline custom_components/huesensor/binary_sensor.py]: Isnt this related to remotes?

Comment (Mariusthvdb) [reply custom_components/huesensor/binary_sensor.py]: yo're referring to line 39? yes, but it is already in the remote.py file. Should it stay in the binary_sensor.py too? Dont think the threshold_offset is for the remotes too, isn't it?

my files work locally as expected without this.

Comment (robmarkcole) [reply custom_components/huesensor/binary_sensor.py]: Yes please remove from binary_sensor file

**Comment by robmarkcole** [comment]
Also can we use the same naming conventions as in the official integration [here](https://github.com/home-assistant/home-assistant/blob/e796de6c599f498a548d50bcd4e8069cbc01529b/homeassistant/components/hue/sensor.py#L60)

**Comment by robmarkcole** [comment]
LGTM

**Pull request merged**